extensions/amp-script/0.1/test/unit/test-user-activation-tracker.js

```diff
@@ -84,7 +80,7 @@ describes.realWin('UserActivationTracker', {}, (env) => {
       return promise
         .then(() => {
           // Skip microtask.
-          return microTask();
+          return macroTask(win.setTimeout);
```

Comment:
I changed `macroTask` and `sleep` to accept a custom `setTimeout`. Some of the tests didn't pass without it, looks like it's a matter of scope (e.g., some test cases pass the `setTimeout` of the inside of an iframe, instead of using the uppermost `setTimeout`. I'm confused as to why there's a difference, but this PR just recreates the existing behaviour of the tests)